src/utils/seafile-error.cpp

```diff
@@ -127,3 +127,44 @@ QString translateSyncErrorCode(const int error_code)
     }
     return error_str;
 }
+
+// The error related to partial persistence may have been resolved.
+// Continuing to display the error could confuse users. Add a description to inform users that this error can be ignored.
```

Comment:
We'll keep file sync error messages in the main window, to inform users that some errors have happened.
But such errors may confuse the users as they don't know how to check them. So we add a tip to the message.